From 178bed3732c0df1ba8ee618cc11213d946804ca9 Mon Sep 17 00:00:00 2001
From: Beginner <70857188+Beginner-Go@users.noreply.github.com>
Date: Thu, 9 Sep 2021 01:47:31 +0800
Subject: [PATCH] dnsproxy: update to 0.39.5

---
 package/lean/dnsproxy/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/lean/dnsproxy/Makefile b/package/lean/dnsproxy/Makefile
index 0c6438620ce9..0e9bfbbaf9e6 100644
--- a/package/lean/dnsproxy/Makefile
+++ b/package/lean/dnsproxy/Makefile
@@ -5,12 +5,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dnsproxy
-PKG_VERSION:=0.39.4
+PKG_VERSION:=0.39.5
 PKG_RELEASE:=$(AUTORELESE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/AdguardTeam/dnsproxy/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=51ab423880141e6ded073f1d5bec7c9e68c3d54df380e75913dbc87aa264237b
+PKG_HASH:=0537c31eb8e9bed29f7fb2645b67dbead3bd3b73149ba20aeca96fcab15321ec
 
 PKG_MAINTAINER:=Tianling Shen <cnsztl@immortalwrt.org>
 PKG_LICENSE:=Apache-2.0
